### PR TITLE
Add SSL support

### PR DESCRIPTION
**Pull request by numansiddique**
Add SSL support

### CODE REVIEW

File: ovn_cluster.sh [hunk 1]
```diff
@@ -298,30 +301,50 @@ function wait-containers() {
...
 # containers.
 function start-db-cluster() {
-    ${RUNC_CMD} exec ${CENTRAL_NAME}-1 ${OVNCTL_PATH} --db-nb-addr=170.168.0.2 --db-nb-create-insecure-remote=yes  \
---db-sb-addr=170.168.0.2 --db-sb-create-insecure-remote=yes --db-nb-cluster-local-addr=170.168.0.2 \
---db-sb-cluster-local-addr=170.168.0.2 start_ovsdb
+    local ssl_certs_path=$1
+    SSL_ARGS=""
+    if [ "$ENABLE_SSL" == "yes" ]; then
+        SSL_ARGS="--ovn-nb-db-ssl-key=${SSL_CERTS_PATH}/ovn-privkey.pem \
+--ovn-nb-db-ssl-cert=${SSL_CERTS_PATH}/ovn-cert.pem \
```
Comment (flavio-fernandes) [inline ovn_cluster.sh]: nit: can we indent this to be inside the if block?

File: ovn_cluster.sh [hunk 2]
```diff
@@ -585,7 +623,7 @@ function start-chassis() {
     ovn_central=no
     ovn_remote=$1
     ovn_add_chassis=$2
-    start $ovn_central $ovn_remote $ovn_add_chassis
+    start $ovn_central $ovn_remote $ovn_add_chassis no
```
Comment (flavio-fernandes) [inline ovn_cluster.sh]: what is that ```no``` for ?

Comment (numansiddique) [reply ovn_cluster.sh]: Removed it. In my initial attempt had thought of generating certificates in start() if "yes" is passed in $4.

Later I changed the approach to generate the ssl certificates during build itself. And I forgot to delete this.

Thanks for catching this.

File: ovn_cluster.sh (hunk 1 above)
Comment (numansiddique) [reply ovn_cluster.sh]: Done

**Pull request merged**